inst/pages/transformation.qmd

```diff
@@ -100,14 +100,18 @@ A complete list of available transformations and parameters, is
 available in the function
 [help](https://microbiome.github.io/mia/reference/transformAssay.html).
 
-## Pseudocount
-
-::: {.callout-warning}
-## TODO
-
-Pseudocount can affect the results. Add here citation to the article. Also
-explain the idea of pseudocount. ancombc2 takes into account the affect by
-performing sensitivity test on multiple pseudocount values.
+::: {.callout-important}
+
+Pseudocount value can have a significant impact on the results when applying 
+a logarithm transformation to normalized data, as the logarithm transformation 
+is a nonlinear operation that can fundamentally change the data distribution. 
+See [Costea et al. 2014](https://www.nature.com/articles/nmeth.2897). 
```

Comment:
Add this to bibliography file and refer to it with [@this_reference]. Check for instance line 68 for reference.

Also add sentence about pseudocount; why it is added

Reply:
Thanks.